### PR TITLE
Bring clarity to campaign and opportunity page navigation

### PR DESCRIPTION
**Pull request by gvn**
Related to #1047

### CODE REVIEW

**Comment by alanmoo** [comment]
Seems like solo pages are having some trouble:

Campaign page centers the header
image

Opportunity page thinks the side nav is still there
image

File: network-api/networkapi/templates/pages/campaign-opportunity-base.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      {% block subcontent %}
+      <div class="{% if page.parent.title != "{{ parentTitle }}" or page.children.all.count > 0 %}col-12 col-md-9{% else %}col-12 col-lg-8 offset-lg-2{% endif %}">
+        <div class="row">
+          <div class="cms col-12">{{ page.landingpage.content | safe }}</div>
+        </div>
+      </div>
+      {% endblock %}
+    </div>
+
+    {% if page.landingpage.signup %}
```
Comment (alanmoo) [inline network-api/networkapi/templates/pages/campaign-opportunity-base.html]: Campaign pages don't have a `signup`, they have `petition`. Should this block be in the landingpage specific template?

**Comment by gvn** [comment]
@alanmoo Ok, I think I got it working by using the old parent title test.

**Comment by gvn** [comment]
Also, for reference, this is how I understand that the pages will be structured in Mezzanine:

```
Opportunity
├── Multipage
│   ├── Sibling 1
│   ├── Sibling 2
│   └── Sibling 3
└── Orphan
Campaigns
├── Multipage
│   ├── Sibling 1
│   ├── Sibling 2
│   └── Sibling 3
└── Orphan
```

File: network-api/networkapi/templates/pages/campaign-opportunity-base.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <div class="row">
+          <div class="cms col-12">{{ page.landingpage.content | safe }}</div>
+        </div>
+      </div>
+      {% endblock %}
+    </div>
+
+
+    <div class="row mb-5">
+      <div class="{% if page.parent.title != parentTitle %}col-md-12{% else %}col-12 col-lg-8 offset-lg-2{% endif %}">
```
Comment (alanmoo) [inline network-api/networkapi/templates/pages/campaign-opportunity-base.html]: Oh, or does changing this to `col-lg-10 offset-lg-1` fix the width issue?

Comment (gvn) [reply network-api/networkapi/templates/pages/campaign-opportunity-base.html]: Yeah, I think that does work better.

**Pull request merged**